### DISCUSSION TITLE
[Agenda] 2023/08/11

### DISCUSSION THREAD

**Discussion by bjwswang** [kubebb/core#233] [Community Meetings]
Please list the items you want to discuss in this meeting with this template!

## Template

```markdown
## [number] Title
/kind propose

### Description

What is this item for?
```

For example:

```markdown
## 1. propose a template for Pull Request
/kind propose

### Description

Here I want to propse a template when making  PR in kubebb projects. 
```

**Comment by bjwswang** [comment]
## 1 `Rating` details
/kind propose

### Description

@0xff-dev pls share the details on how we implement this `Rating` based on our offline discussions.

**Comment by Lanture1064** [comment]
## 2 Stable Diffusion
/kind propose

### Description

Sharing some info about SD, including installation, usage, deployment and other tips and experience.
